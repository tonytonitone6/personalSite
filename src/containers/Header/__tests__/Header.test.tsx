
describe('123', () => {
  test('test render', () => {
    expect(true).toBe(true);
  })
})