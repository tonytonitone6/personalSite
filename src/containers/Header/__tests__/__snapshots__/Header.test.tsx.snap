// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render icon item renders 1`] = `ShallowWrapper {}`;

exports[`render icon item return the default empty array 1`] = `ShallowWrapper {}`;
